src/utilities/theme/color-adjustments.ts

```diff
@@ -590,7 +592,7 @@ export const colorAdjustments: ColorAdjustments = {
   },
   highlightText: {
     baseColor: 'highlight',
-    light: {lightness: 98},
+    light: {lightness: 38},
```

Comment:
I'm not sure about leaving this change as part of this PR but this value was well below our contrast standards.

Before | After
---|---
<img width="640" src="https://screenshot.click/Playground__Playground_-_Playground__Storybook_2019-11-19_12-28-57.png"> | <img width="640" src="https://screenshot.click/Playground__Playground_-_Playground__Storybook_2019-11-19_12-29-10.png">

Reply:
Looking at the banner and how this color is used I am not sure why the color is white. I might be missing some context. Best to wait for @tmlayton or @danrosenthal.

Reply:
This change seems fine, the lightness of the text for similar roles in 30-40 in light mode and 60-70 in dark mode. They should probably be consistent across the board 30/70, 40/60, or something else.